R/item_list_files.R

```diff
@@ -70,17 +75,44 @@ item_list_files = function(sb_id, recursive = FALSE, ..., session=current_sessio
 		files <- item$files
 	}
 	
-	out <- data.frame(stringsAsFactors = FALSE)
+	files <- c(files, get_facet_files(item))
+	
+	lf <- length(files)
+	
+	out <- data.frame(fname = rep("", lf),
+										size = rep(NA_integer_, lf),
+										url = rep("", lf),
+										facet = rep("", lf))
```

Comment:
I know that this code will cause rebuilds in my pipelines because of adding the column `facet` even when there aren't facets. I am fine accepting that but wanted to raise the question about whether the `facet` column should exist at all in the output of `item_list_files()` if there are no facets in the item?

Reply:
Good question. I tend toward having all possible names present even if they will be null, but I see what you mean about it causing change when change isn't needed. I could go either way...

Reply:
I could go either way, too, so we can leave with how it is already implemented. Might be worth noting the change to the output of `item_list_files()` in the release notes is all :)

Reply:
OK -- that's a good path. I'll make sure it's noted in the NEWS